### PR TITLE
Register Mantra mainnet and Dukong testnet

### PR DESCRIPTION
**Pull request by alex-stmr**

### CODE REVIEW

File: testnets/mantradukongtestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "mantradukongtestnet",
```
Comment (JeremyParish69) [inline testnets/mantradukongtestnet/assetlist.json:3]: the testnet chain_name must be "mantrachaintestnet"

File: testnets/mantradukongtestnet/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                    "denom": "om",
+                    "exponent": 6
+                }
+            ],
+            "type_asset": "sdk.coin",
+            "base": "uom",
+            "name": "MANTRA Chain Dukong",
+            "display": "om",
+            "symbol": "OM",
+            "coingecko_id": "mantra-dao",
```
Comment (JeremyParish69) [inline testnets/mantradukongtestnet/assetlist.json:23]: Please remove the cgid here

File: testnets/mantradukongtestnet/assetlist.json (hunk 1 above)
Comment (alex-stmr) [reply testnets/mantradukongtestnet/assetlist.json:3]: Thanks for the review @JeremyParish69 
What would you recommend for the name as `mantrachaintestnet` is already taken by the first testnet of Mantra (Hongbai)

cf: https://github.com/cosmos/chain-registry/blob/d7165465d3758d9af3f846e4f68c765dd946e45e/testnets/mantrachaintestnet/chain.json#L3

Dukong is the second one

File: testnets/mantradukongtestnet/assetlist.json (hunk 2 above)
Comment (alex-stmr) [reply testnets/mantradukongtestnet/assetlist.json]: Fixed 👍

File: testnets/mantradukongtestnet/assetlist.json (hunk 1 above)
Comment (JeremyParish69) [reply testnets/mantradukongtestnet/assetlist.json:3]: My apologies, I assumed it wasn't already taken. In that case, the name should be `mantrachaintestnet2`

File: testnets/mantrachaintestnet2/chain.json [hunk 3]
```diff
@@ -0,0 +1,146 @@
+{
+    "$schema": "../../chain.schema.json",
+    "chain_name": "mantradukongtestnet",
```
Comment (JeremyParish69) [inline testnets/mantrachaintestnet2/chain.json:3]: this value has to also be updated to mantrachaintestnet2

Comment (alex-stmr) [reply testnets/mantrachaintestnet2/chain.json]: My mistake, I just updated it, thanks for catching 🙏

**Pull request merged**